packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -562,18 +567,14 @@ Possible alternatives for such technical user authentication are BasicAuthentica
     }
   }
 
-  private selectSubscriberJwt(): JwtPayload {
-    if (!this.subscriberToken) {
+  private selectSubscriberJwt(): JwtPair {
```

Comment:
@FrankEssenberger I have changed this function and I think logically my changes are the same for the old state. This is used for the cache key, and I guess this is not correct for the combination case (iss and jwt), because both tokens are relevant I guess. Do you agree?